### DISCUSSION TITLE
Floral Fabrics. [Zhixin Zhong]

### DISCUSSION THREAD

**Discussion by qinghe1001** [realitydeslab/playshop-summer-2024#13] [Proposal]
### Research Question: 
How can we help plants communicate, understand their true intentions, perform pollination, and protect the ecosystem in the future, exploring ways to facilitate plant interaction and promote ecological balance?
我们如何在未来帮助植物交谈、理解植物的真实意图，进行授粉和保护生态，探索植物间的交流和促进生态平衡的方式？

### Keywords: 
Plant communication, ecological protection, human pollination, plant diversity
植物交流，生态保护，人类授粉，植物多样性

### Abstract
By 2050, some pollinator insect species may face significant population and habitat declines, and humans will take on the role of pollinators, becoming part of plant communication. Future ecological protection is not only a human responsibility but a result of collaboration between plants and nature. By understanding and promoting plant communication and human-assisted pollination, we can create a more balanced and harmonious ecosystem. Plants "talk" through chemical signals released by their roots and into the air, which not only affect plant growth and health but also play a crucial role in the balance of the entire ecosystem. Using high-tech equipment and biotechnology, we can amplify plant communication signals, making them easier for other plants and humans to perceive. At the same time, humans will wear pollination clothing and directly participate in plant pollination. In this environment, plants can more effectively warn each other of pests and diseases, exchange pollen for more communication, and enhance the resilience and adaptability of the entire ecosystem. By interpreting plant language, humans will also be guided by plant intentions in their propagation efforts. Through human propagation activities, the ecosystem will be reshaped.
2050年，一些授粉昆虫物种可能会面临严重的种群和栖息地减少，人类将承担起授粉者这一职责，成为植物交流的一部分。未来的生态保护不仅仅是人类的责任，更是植物与自然共同协作的结果。通过理解和促进植物间的交流以及人类的辅助授粉，创造一个更平衡、更和谐的生态系统。植物通过根系和空气中释放的化学信号进行“对话”，这些信号不仅影响着植物的生长和健康，还对整个生态系统的平衡起着至关重要的作用。通过高科技设备和生物技术，我们可以放大植物的交流信号，使其更易被其他植物和人类感知。同时，人类将通过穿戴授粉服装，直接参与到植物授粉过程中。在这种环境下，植物将能够更有效地互相警示害虫和疾病，通过交换花粉来达成更多的交流，增强整个生态系统的韧性和适应能力。同时通过解读植物的语言，也将引导人类参考植物的意图进行传播。在人类的传播行为下，生态也将被重塑。

### Implementation Design
**Pollination Clothing:**
Design clothing with pollen capture and release devices, such as biomimetic small brushes and microporous membranes. The pleats will expand and contract with the wearer's breathing, allowing the wearer to naturally perform pollination during daily activities.
授粉服装：
设计带有花粉捕捉和释放装置的服装，如仿生学设计的微小毛刷和微孔膜，褶皱随着穿戴者的呼吸张合，使佩戴者在日常活动中自然进行授粉。

**Visualization and Greeting Cards:**
Edit the audio produced during plant pollination and visualize it into greeting cards, coated with pollen. During the exchange of greeting cards, pollen will also be exchanged.
可视化与贺卡：
将植物授粉时产生的音频编辑下来，可视化成贺卡。并涂上花粉。在贺卡传递的过程中也会进行花粉之间的交换。

我们是否能理解植物的语言和授粉？
Can we understand the language of plants and perform pollination?

### Visual Reference / Illustrations
![Screenshot_2024-06-28-12-21-18-06_a2db1b9502c98f2](https://github.com/realitydeslab/playshop-summer-2024/assets/173800345/dd05efd2-9af5-464b-95a9-238927d954a8)
![Screenshot_2024-06-28-12-21-52-04_a2db1b9502c98f2](https://github.com/realitydeslab/playshop-summer-2024/assets/173800345/72ef5248-4839-4cea-b5b0-8e8bd6d84a70)
![Screenshot_2024-06-28-12-24-08-21_a2db1b9502c98f2](https://github.com/realitydeslab/playshop-summer-2024/assets/173800345/14b14308-68c8-4724-b57f-1628eacd9efb)
by CHAVARRI Spring/Summer Collection
### Reading List
Johnson, L. E. (1993). Morally deep world: An essay on moral significance and environmental ethics. Cambridge University Press.

### References:
 [1]:Yang, A. (2023, April 12). Plants can talk. Yes, really. Here’s how. National Geographic. Retrieved from https://www.nationalgeographic.com/science/article/plants-can-talk-yes-really-heres-how
 [2]:Biesmeijer, J. C., Roberts, S. P. M., Reemer, M., & Kunin, W. E. (2006). Parallel declines in pollinators and insect-pollinated plants in Britain and the Netherlands. Science, 313(5785), 351-354. https://doi.org/10.1126/science.1127863
 [3]:Nath, R., Singh, H., & Mukherjee, S. (2022). Insect pollinators decline: an emerging concern of Anthropocene epoch. Journal of Apicultural Research, 62(1), 23-38. https://doi.org/10.1080/00218839.2022.2088931
 [4]:University of Cambridge. (n.d.). Pollinators: first global risk index for species declines and effects on humanity. Retrieved from https://www.cam.ac.uk/stories/pollinatorsriskindex
 [5]:Hiernaux, Q. (2021). The ethics of plant flourishing and agricultural ethics: Theoretical distinctions and concrete recommendations in light of the environmental crisis. Philosophies, 6(4), 91. https://doi.org/10.3390/philosophies6040091
 [6]:Khait, I., Lewin-Epstein, O., Sharon, R., Sade, N., Yovel, Y., & Hadany, L. (2023). Sounds emitted by plants under stress are airborne and informative. Cell, 186(7), P1328-1336.E10. https://doi.org/10.1016/j.cell.2023.03.009
 [7]:Hiernaux, Q. (2021). The Ethics of Plant Flourishing and Agricultural Ethics: Theoretical Distinctions and Concrete Recommendations in Light of the Environmental Crisis. Philosophies, 6(4), 91. https://doi.org/10.3390/philosophies6040091

**Comment by qinghe1001** [comment]
https://www.media.mit.edu/projects/elowan-a-plant-robot-hybrid/overview/

**Comment by qinghe1001** [comment]
PlantWave measures microfluctuations in conductivity between two points on a plant. The amount of conductivity between these two points is largely related to how much water there is between them. That changes as plants photosynthesize and move chloroplasts around.
The variation is graphed over time, which produces a wave. We take that wave and translate it into pitch. Then those pitch messages are routed to control instruments we designed. So, each note you hear is an expression of a change happening within the plant in the moment. The greater the distance between two adjacent notes, the greater change is happening within the plant.
https://www.youtube.com/watch?v=w0tNEp4m5PY

**Comment by qinghe1001** [comment]
https://pollinator.art/

**Comment by qinghe1001** [comment]
https://www.mdpi.com/journal/philosophies/special_issues/VFBOF9122W

**Comment by qinghe1001** [comment]
Key Frame
![quality_restoration_20240702121830605](https://github.com/realitydeslab/playshop-summer-2024/assets/173800345/8dc1cc91-cc85-44f7-aefe-5453c11cb0f8)

**Comment by qinghe1001** [comment]
Tagline: Declaration of Plant Rights: Sunlight, Air, Water, and Your Respect!
Video clip scenario:"If there is endless debate, let's listen to what the plants have to say."

**Reply by glitch-kraft** [reply]
**Title**
Floral Fabrics. 

**Tagline**
Floral Fabrics. Cross-pollination and the future of wearable flora. But what about plants' rights to sunlight, air, water and freedom of mobility?
